src/js/components/FormField/FormField.js

```diff
@@ -473,17 +473,7 @@ const FormField = forwardRef(
             ? (event) => {
                 event.persist();
                 if (onChange) onChange(event);
-                if (contextOnChange) {
-                  const debouncedFn = debounce(() => {
-                    contextOnChange(event);
-                    // A half second (500ms) debounce can be a helpful starting
-                    // point. You want to give the user time to fill out a
-                    // field, but capture their attention before they move on
-                    // past it. 2 second (2000ms) might be too long depending
-                    // on how fast people type, and 200ms would be an eye blink
-                  }, 500);
-                  debouncedFn();
-                }
+                if (contextOnChange) setDebounced(() => contextOnChange);
```

Comment:
We seem to have lost the event argument to `contextOnChange()`

Reply:
Fixed in latest commit